dali/test/python/test_operator_join.py

```diff
@@ -75,17 +78,17 @@ def test_cat_empty_input():
     pipe = dali.pipeline.Pipeline(batch_size=1, num_threads=3, device_id=0)
     with pipe:
         src1 = dali.types.Constant(np.array(
-            [[1, 2, 3 ,4],
-             [5, 6, 7, 8],
-             [9,10,11,12]]))
+            [[1, 2,  3,  4],
+             [5, 6,  7,  8],
+             [9, 10, 11, 12]]))
```

Comment:
Normally, numbers are aligned to the right or to the decimal point.
```suggestion
            [[1,  2,  3,  4],
             [5,  6,  7,  8],
             [9, 10, 11, 12]]))
```